### DISCUSSION TITLE
Frage zu Zeitplänen

### DISCUSSION THREAD

**Discussion by sirius-black-2** [camueller/SmartApplianceEnabler#393] [Allgemein]
Hallo zusammen,

irgendwie komme ich mit den Zeitplänen nicht klar...
Beispiel Waschmaschine: Die muss laufen und zwar mind. 2h (als Tagesplan, mit Laufzeit und allen Wochentagen)
Die minimale Laufzeit also 2h (soll ja einmal gestartet durchlaufen), max. 5 h im Zeitfenster 06:00 - 17:00
Jetzt wurde die Waschmaschine um kurz vor sieben gestartet, vom SHM/SAE unterbrochen, aber leider bis 15:00 nicht wieder gestartet, (danach über die Ampel eingeschaltet).
Im Sunny Portal sieht es so als hätte der SHM ihn von 06:57 (erstes einschalten) bis 09:57 eingeplant. Ein Balken erscheint in der Verbraucherübersicht (bein drübersrollen erscheint Waschmaschine: Zeitfenster 06.57 - 09.57, Verbrauch 0 Wh) um kurz nach 15.00 (manueller Start) kommt wieder ein Balken (Waschmaschine: Zeitfenster 15.03 - 17.15 Verbrauch 151 Wh). 
Die Priorität im Sunny Portal ist auf hoch. Ich habe jetzt mal den Schieberegler mehr in Richtung Netzbezug gestellt, da bei diesem bewölktem Wetter nicht viel Leistung runterkommt, aber eigentlich sollte die Waschmaschine ja auf jeden Fall staten. Für Anregungen bin ich dankbar und die Anleitung hier zu den Zeitplänen habe ich natürlich auch gelesen. Ich habe auch geprüft, ob auf meinem Raspi evtl. die Uhrzeit falsch eingestellt war.
![LZ_WM](https://user-images.githubusercontent.com/99977252/204632564-2aa27efe-a2bf-42a7-98cb-41431e38fb59.PNG)
![SAE_Zeitplan](https://user-images.githubusercontent.com/99977252/204632760-906abae6-db77-4901-aac9-cc208e86cf2b.PNG)

**Comment by camueller** [comment]
Du solltest keine Mindest-Laufzeit angeben. Dir sollte nur gesetzt werden zur Verwendung von Überschussenergie, was bei Geschirrspüler oder Waschmaschine nicht der Fall ist.

Siehe https://github.com/camueller/SmartApplianceEnabler/blob/master/doc/Schedules_DE.md
